packages/ethereum/contracts/HoprWrapper.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.6.0;
+
+
+import "@openzeppelin/contracts/introspection/IERC1820Registry.sol";
+import "@openzeppelin/contracts/introspection/ERC1820Implementer.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+import "@openzeppelin/contracts/token/ERC777/IERC777Recipient.sol";
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/utils/ReentrancyGuard.sol";
+import "./HoprToken.sol";
+
+
+/**
+ * HoprWrapper is a smart contract that may receive xHOPR (ERC-677)
+ * and it will send back wxHOPR (ERC-777) and vice versa.
+ */
+contract HoprWrapper is IERC777Recipient, ERC1820Implementer, ReentrancyGuard {
+    using SafeMath for uint256;
+    using SafeERC20 for IERC20;
+
+    // setup ERC1820
+    IERC1820Registry internal constant _ERC1820_REGISTRY = IERC1820Registry(0x1820a4B7618BdE71Dce8cdc73aAB6C95905faD24);
+    bytes32 public constant TOKENS_RECIPIENT_INTERFACE_HASH = keccak256("ERC777TokensRecipient");
+
+    // xHOPR token
+    IERC20 public xHOPR;
+    // wrapped xHOPR address
+    HoprToken public wxHOPR;
+
+    // total amount of xHOPR collected in this contract
+    uint256 total_xHOPR = 0;
```

Comment:
Why do we need to store the xHOPR balance of the wrapper contract?

Reply:
no need - removing